.travis.yml

```diff
@@ -0,0 +1,26 @@
+sudo: false
+
+language: php
+
+cache:
+  directories:
+    - $HOME/.composer/cache/files
+
+php:
+  - 5.6
+  - 7.0
+  
+matrix:
+  fast_finish: true
+  include:
+    - php: 7.0
+      env: deps=low
+
+before_script:
+  - phpenv config-rm xdebug.ini
+  - composer self-update
+  - if [[ "$deps" = "low" ]]; then composer update --prefer-lowest --prefer-stable; else composer install --prefer-dist; fi
```

Comment:
`--prefer-dist` shouldn't be necessary anymore: https://github.com/composer/composer/issues/4884#issuecomment-195229989